docs/config.md

```diff
@@ -267,6 +267,15 @@ a `$`):
 
 `less -FRX` is the default pager in the absence of any other setting.
 
+Additionally, paging behavior can be toggled via `ui.paginate` like so:
+
+```toml
+# Enable pagination where required (default) 
```

Comment:
"required" sounds a bit too strong. Maybe "Enable pagination for commands that support it"